bin/crypto_conditions_test_gen/conditions.py

```diff
@@ -0,0 +1,1217 @@
+#!/usr/bin/env python
+
+# This script was run using python 3.6
+# When using anaconda python, the following additional packages were installed (using conda install XXX):
+# pyasn1 (conda install pyasn1)
+# cryptography (conda install cryptography)
+# nacl (conda install -c conda-forge pynacl)
+
+import base64
+import binascii
+import codecs
+
+import collections
+from collections import defaultdict
+
+import cryptoconditions_asn1
+
+import cryptography
+from cryptography.hazmat.backends import default_backend
+from cryptography.hazmat.primitives.asymmetric import rsa
+from cryptography.hazmat.primitives import serialization
+
+import itertools
+
+import known_signing_keys
+
+import nacl
+import nacl.encoding
+import nacl.hash
+import nacl.signing
+
+import logging
+
+import pyasn1
+from pyasn1.type import univ
+from pyasn1.codec.der.decoder import decode
+from pyasn1.codec.der.encoder import encode
+
+import string
+
+from IPython.core.debugger import Tracer
+
+logging.basicConfig(filename='conditions.log', level=logging.DEBUG)
+condition_logger = logging.getLogger('condition')
+
+condition_test_template_prefix = \
+'''
```

Comment:
Should we add a comment at the top indicating that the file was generated and "DO NO EDIT"? Or are we okay with people editing the generated sources?